packages/template-cli/templates/src/ssr/config/theme.ts

```diff
@@ -1,20 +1,53 @@
-export type Theme = {
-    [key: string]: string;
-};
+import {createMuiTheme} from "@material-ui/core/styles"
+import {red} from "@material-ui/core/colors"
 
-export const theme: Theme = {
-    gutter: "1rem",
-    borderRadius: "0.5rem",
-    darkestColor: "#212121",
-    amidoColor: "#fecb07",
-    minimumPadding: "0.5rem"
-};
+// Create a theme instance.
+const theme = createMuiTheme({
+    palette: {
+        primary: {main: "#000000"},
+        secondary: {main: "#FECB07"},
+        error: {
+            main: red.A400,
+        },
+        background: {
+            default: "#fff",
+        },
+    },
+    overrides: {
+        MuiButton: {
+            root: {
+                borderRadius: "5px",
+                fontSize: "13px",
+                textTransform: "none",
+            },
+            contained: {
+                boxShadow: "0px",
+                textTransform: "uppercase",
+            },
+        },
+        MuiListItem: {
+            root: {
+                marginTop: "20px",
+            },
+        },
+    },
+    typography: {
+        fontFamily: ["Arial", "Work sans"].join(","),
+        h2: {
```

Comment:
This was changed to define h1 onwards when we first did the theme. We have now regressed the typography.